VCFv4.4.draft.tex

```diff
@@ -1257,10 +1248,10 @@ \subsubsection{Sample mixtures}
 \begin{flushleft}
 \begin{tabular}{ l l l l l l l l l l l }
 \#CHROM & POS & ID & REF & ALT & QUAL & FILTER & INFO & FORMAT & Blood & TumorSample \\
-2 & 321681 & bnd\_W & G & G$]2:421681]$ & 6 & PASS & SVTYPE=BND;MATEID=bnd\_U & GT:CNADJ & 0:1 & 1:1 \\
-2 & 321682 & bnd\_V & T & $[2:421682[$T & 6 & PASS & SVTYPE=BND;MATEID=bnd\_X & GT:CNADJ & 0:1 & 1:1 \\
-13 & 421681 & bnd\_U & A & A$]2:321681]$ & 6 & PASS & SVTYPE=BND;MATEID=bnd\_W & GT:CNADJ & 0:1 & 1:1 \\
-13 & 421682 & bnd\_X & C & $[2:321682[$C & 6 & PASS & SVTYPE=BND;MATEID=bnd\_V & GT:CNADJ & 0:1 & 1:1 \\
+2 & 321681 & bnd\_W & G & G$]2:421681]$ & 6 & PASS & ;MATEID=bnd\_U & GT:CNADJ & 0:1 & 1:1 \\
+2 & 321682 & bnd\_V & T & $[2:421682[$T & 6 & PASS & ;MATEID=bnd\_X & GT:CNADJ & 0:1 & 1:1 \\
+13 & 421681 & bnd\_U & A & A$]2:321681]$ & 6 & PASS & ;MATEID=bnd\_W & GT:CNADJ & 0:1 & 1:1 \\
+13 & 421682 & bnd\_X & C & $[2:321682[$C & 6 & PASS & ;MATEID=bnd\_V & GT:CNADJ & 0:1 & 1:1 \\
```

Comment:
Here and in multiple other locations, an extra semicolon is present. E.g. `;MATEID=bnd\_U`, which should be `MATEID=bnd\_U`.